### DISCUSSION TITLE
Netmiko cisco asa send config set gives "'Failed to enter configuration mode"

### DISCUSSION THREAD

**Discussion by akshayavk96** [ktbyers/netmiko#3128] [Q&A]
Hello, 
I am trying to send a list of commands to configuration mode using netmiko send_config_set function on cisco ASA. However I keep on getting ('Failed to enter configuration mode.',). I tried setting up enable mode, adding delay facto to send_config_set, but no luck. Please advice.
hostname = "x.x.x.x"

device_type = "cisco_asa"
net_connect = ConnectHandler(
            ip=hostname,
            device_type=device_type,
            username=user_name,
            password=password,
            global_delay_factor=4,
            port=port,
            banner_timeout=200,
            conn_timeout=30)
conf = ["crypto ca trustpoint SSL-Trustpoint-test", "enrollment terminal"]
if not net_connect.check_enable_mode():
    c3 = net_connect.enable()
c1 = net_connect.send_config_set(conf , delay_factor=30)

Error:
2023-02-23 14:00:52,016[ERROR ('Failed to enter configuration mode.',)

**Comment by ktbyers** [comment]
@akshayavk96 Can you send over the full exception stack trace?
